galaxy_ng/tests/functional/api/__init__.py

```diff
@@ -0,0 +1,2 @@
+# coding=utf-8
```

Comment:
Minor: For python 3 source file encoding is utf-8 by default, therefore this comment is redundant.